pkg/server/plugin/notifier/k8sbundle/k8sbundle.go

```diff
@@ -138,8 +144,7 @@ func (p *Plugin) Configure(ctx context.Context, req *spi.ConfigureRequest) (resp
 		config.ConfigMapKey = defaultConfigMapKey
 	}
 
-	p.setConfig(config)
-	return &spi.ConfigureResponse{}, nil
+	return &spi.ConfigureResponse{}, p.setConfig(config)
```

Comment:
nit: even though gRPC will handle it properly, we shouldn't return a non-nil response with an error. I'd suggest splitting this up and returning an explicit `nil, err` if `setConfig` fails.

Reply:
done